src/poker/game.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
 import random
-from cardecky import Deck, HandRanker
+from src.poker.cardecky import Deck, HandRanker
```

Comment:
Have you been able to run this without using src.poker as a prepend to the cardecky.py file? I'm not quite clear why it gives me issues but none of the files work for me at all using this syntax

Reply:
you're right, fixing it now